.github/workflows/iroha2-dev.yml

```diff
@@ -64,23 +64,23 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-          repository: ${{ secrets.DOCKERHUB_ORG }}/iroha
+          repository: hyperledger/iroha2
```

Comment:
Could you migrate action from `1.1.0` to `2.0`? It is done like that in `iroha2-dev-pr.yml`